### ISSUE TITLE
[windows] Moving window to other screen with "win+shift+right" dpesm

### ISSUE THREAD

**Issue by Serock3** [alacritty/alacritty#8108]
For bug reports, the following information can help speed up the process. Please
describe the bug that you have found and what you would expect to happen
instead.

### System

OS: Windows 11
Version: 0.13.1 (fe2a3c5)

### Logs

Keyboard and bindings: 
[Alacritty-13996.log](https://github.com/user-attachments/files/16437743/Alacritty-13996.log)

**Comment by chrisduerr** [comment]
So it works when the monitor has the same scale? It also works with different applications?

**Comment by Serock3** [comment]
Yes to both. Switching Alacritty to another monitor works fine when the scale is the same, but every other application I tested can switch normally regardless of scale difference.

**Comment by chrisduerr** [comment]
@kchibisov Do you know who maintains winit's Windows backend? This sounds like a Microsoft bug, but I don't know if winit needs to implement this itself.

**Comment by kchibisov** [comment]
it's @notgull for now.
